app/presenters/organisations/index_presenter.rb

```diff
@@ -40,24 +37,21 @@ def works_with(organisation)
       end
     end
 
-  private
+    def executive_office?(organisation_type)
+      organisation_type.eql?(:number_10)
```

Comment:
It's possible that some of the code here is going to duplicate some of the code for the org pages themselves, I wonder if there's a good way around that or if a little duplication is to be expected?

Reply:
I think for now we should leave this where it is and address any potential duplication on the org pages as it arises